modules/tinymce/src/themes/silver/main/ts/ui/dialog/WindowManager.ts

```diff
@@ -244,6 +246,134 @@ const setup = (extras: WindowManagerSetup): WindowManagerImpl => {
+            inlineDialog.clear();
+            closeWindow(dialogUi.instanceApi);
+          }
+        },
+        extras.backstages.popup,
+        ariaAttrs
+      );
+
+      const inlineAdditionalBehaviours = (editor: Editor): Behaviour.NamedConfiguredBehaviour<any, any>[] => {
+        return [
+          Docking.config({
+            contextual: {
+              lazyContext: () => Optional.some(Boxes.box(SugarElement.fromDom(editor.getContentAreaContainer()))),
+              fadeInClass: 'tox-dialog-dock-fadein',
+              fadeOutClass: 'tox-dialog-dock-fadeout',
+              transitionClass: 'tox-dialog-dock-transition'
+            },
+            modes: [ 'top', 'bottom' ],
+            lazyViewport: (comp) => {
+              const optScrollingContext = ScrollingContext.detectWhenSplitUiMode(editor, comp.element);
+              return optScrollingContext
+                .map(
+                  (sc) => {
+                    const combinedBounds = ScrollingContext.getBoundsFrom(sc);
+                    return {
+                      bounds: combinedBounds,
+                      optScrollEnv: Optional.some({
+                        currentScrollTop: sc.element.dom.scrollTop,
+                        scrollElmTop: SugarLocation.absolute(sc.element).top
+                      })
+                    };
+                  }
+                ).getOrThunk(
+                  () => {
+                    return {
+                      bounds: Boxes.win(),
+                      optScrollEnv: Optional.none()
+                    };
+                  }
+                );
+            }
+          })
+        ];
+      };
+
+      const inlineDialogComp = GuiFactory.build(InlineView.sketch({
+        lazySink: extras.backstages.popup.shared.getSink,
+        dom: {
+          tag: 'div',
+          classes: [ ]
+        },
+        // Fires the default dismiss event.
+        fireDismissalEventInstead: { },
+        ...isToolbarLocationTop ? { } : { fireRepositionEventInstead: { }},
+        inlineBehaviours: Behaviour.derive([
+          AddEventsBehaviour.config('window-manager-inline-events', [
+            AlloyEvents.run(SystemEvents.dismissRequested(), (_comp, _se) => {
+              AlloyTriggers.emit(dialogUi.dialog, formCancelEvent);
+            })
+          ]),
+          ...inlineAdditionalBehaviours(editor)
+        ]),
+        // Treat alert or confirm dialogs as part of the inline dialog
+        isExtraPart: (_comp, target) => isAlertOrConfirmDialog(target)
+      }));
+      inlineDialog.set(inlineDialogComp);
+
+      const getInlineDialogBounds = (): Optional<Boxes.Bounds> => {
+        return extras.backstages.popup.shared.getSink().toOptional().bind((s) => {
+          const optScrollingContext = ScrollingContext.detectWhenSplitUiMode(editor, s.element);
+
+          // margin bottom of the screen/editor content area container
+          const margin = 15;
```

Comment:
Can this margin be changed by the integrators such as via options? I feel that the property should be declared outside as a const like [this](https://github.com/tinymce/tinymce/blob/c12fb6af3b6b72886946dd3fb1bb1b8be249949f/modules/tinymce/src/plugins/charmap/main/ts/ui/Dialog.ts#L48)

Reply:
I'm unsure about having the margin as an option for the integrators, like the current inline dialogs. I believe they could customize the CSS margins if they wish to have more margins. 

And for the constant variable, remember there was a discussion in a private repo to keep it camel case. 😵‍💫 https://github.com/tinymce/tinymce/pull/8205#discussion_r1003873601